modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -154,6 +136,40 @@ The policy must be in the Stable state to upload the implementation file in a la
 
 To implement the policy so that it is available at runtime, follow the steps in xref:add-policy-implementation-files-to-published-policy-definition.adoc[Add Policy Implementation Files to Published Policy Definition].
 
+== Create a GraphQL API
+A GraphQL API is a specification of a service that defines a schema, which is the collection of object definitions. These objects are used when interacting with the API, either as input data or part of the API responses. The GraphQL API allows you to query the API instead of accessing operation that returns a fixed definition of data.
```

Comment:
```suggestion
A GraphQL API is a specification of a service that defines a schema, which is the collection of object definitions. These objects are used when interacting with the API, either as input data or as part of the API responses. The GraphQL API allows you to query the API instead of accessing an operation that returns a fixed definition of data.
```

Reply:
Thanks, fixed